en/user_guide/content.rst

```diff
@@ -0,0 +1,58 @@
+Contents
+========
+
+Contents are managed in the Back Office.
+
+
+Listing
+-------
+Contents listing is accessed in the Editorial section, by clicking on the "Contents" link. A submenu
+lists all Content Types availables.
+
+.. image:: ../../images/content_list_submenu.png
+
+By clicking a Content Type, the list of related contents appears.
+
+.. image:: ../../images/content_list.png
+
+This list has a pagination and filters, and shows the name, the status, the version and the language
+of existing contents. From this list, new content can be created, and existings ones can be edited or
+deleted.
+
+
+Creation
+--------
+From the content list, creation form can be access by clicking on the "Add" button. Informations asked
+depend on the Content Type, each Content Type has its own form. By clicking on the "Save" button, the
+content is created, with the version number 1 and the initial status of the workflow, aka "Draft" in a
+fresh install.
+
+
+Edition
+-------
+From the content list, clicking on an "Edit" button make an edit form appear. The edit form is mainly
+the same as the creation form, completed with current values, but has three more functionnalities.
+
+.. image:: ../../images/content_edit.png
+
+* **Status**: on the upper right corner of the form, a button allows you to change the workflow
```

Comment:
If the request has been taken in account, can you suppress the comment, it helps the readability of the page